plenum/common/event_bus.py

```diff
@@ -24,11 +24,18 @@ def __init__(self, send_handler: SendHandler):
         self._send_handler = send_handler
         self._recv_handlers = InternalBus()
 
+        # list of connected nodes
+        self._connecteds = []
+
+    @property
+    def connecteds(self):
+        return self._connecteds
+
     def subscribe(self, message_type: Type, recv_handler: RecvHandler):
         self._recv_handlers.subscribe(message_type, recv_handler)
 
-    def send(self, message: Any, dst: Destination = None):
-        self._send_handler(message, dst)
+    def send(self, message: Any, dst: Destination = None, recipients: List=None):
```

Comment:
I believe `dst: Destination` is sufficient and `recipients` are not needed since `Destination ` is already a uinion supporting broadcast, sending to one or many nodes.